gensrc/thrift/BackendService.thrift

```diff
@@ -144,4 +161,6 @@ service BackendService {
     TTabletStatResult get_tablet_stat();
 
     Status.TStatus submit_routine_load_task(1:list<TRoutineLoadTask> tasks);
+    // This is used for getting some information via Backend
+    TProxyResult get_info(1:TProxyRequest request);
```

Comment:
Actually, new interface should use Brpc, and old interface should be changed to Brpc.
Because Brpc is better for example it support async, connection reused

Reply:
OK, I will change it